src/Bicep.Core.UnitTests/Assertions/StringAssertionsExtensions.cs

```diff
@@ -60,5 +61,15 @@ public static AndConstraint<StringAssertions> EqualWithLineByLineDiffOutput(this
 
             return new AndConstraint<StringAssertions>(instance);
         }
+
+        public static AndConstraint<StringAssertions> BeEquivalentToIgnoringNewlines(this StringAssertions instance, string expected, string because = "", params object[] becauseArgs)
+        {
+            var normalizedActual = Regex.Replace(instance.Subject, "\r?\n", "\n");
+            var normalizedExpected = Regex.Replace(expected, "\r?\n", "\n");
```

Comment:
I think `\r?\n` doesn't match `\r`? Also there's `StringUtils.ReplaceNewlines` that can be used.